css/style.css

```diff
@@ -16,4 +12,259 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* General styles */
+
+:root {
+    
+    --grey: #616668;
+}
+
+html, body {
+    font-family: 'Roboto', sans-serif;
+    -webkit-font-smoothing: antialiased;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    width: 100%;    
+    font-family: Roboto, sans-serif; 
+    overflow-x: hidden;  
+}
+
+main {   
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-top: 20px;
+}
+
+/* img{
+    max-width: 100%;
+    height: auto;
+    display: block;
+} */
+
+/* Navigation */
+
+nav {
+    display: flex;   
+    justify-content: space-between;
+    align-items: center;
+    box-sizing: border-box;
+    padding: 20px 40px;
+    height: 90px;
+}
+
+nav ul {
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+    font-size: 1.35rem;
+    font-weight: 500;
+    cursor: pointer;      
+}
+
+nav ul li {
+    padding: 0 20px;    
+}
+
+nav ul li a {
+    text-decoration: none;
+    color: #757e83;
+}
+
+nav ul li a:hover {
+    color: var(--orange);
+}
+
+.active {
+    color: black;
+}
+
+#hamburger {
+    display: none;
+}
+
+/* Hero */
+
+.hero {
+    background-color: black; 
+    background-image: url(../img/first-background.jpg);
+    color: white;
+    background-repeat: no-repeat;
+    background-size: 162%;
+    background-position: center top 30%;
+    min-height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center; 
+    font-size: 2.25rem; 
+    filter:saturate(0.7);     
+}
+
+.hero-text-and-button p {
+    font-weight: 300;
+}
+
+/* Typography */
+
+h1 {
+    font-weight: 200;
+    margin-bottom: 30px;
+    padding-top: 80px;
+}
+
+h1, h2, h3 {
+    text-align: center;    
+}
+
+h2 {
+    font-size: 3.5rem;
+    font-weight: 300;
+    padding-top: 60px;
+}
+
+h3 {
+    font-size: 2rem;
+    font-weight: 300;    
+}
+
+
+/* Button */
+
+button {
+    background-color: #E0633A;  
+    font-size: 1.4rem;
+    font-weight: 400;
+    padding: 22px 45px;
+    border-style: none;
+    border-radius: 6px;
+    display: block;
+    margin: 0 auto;
+    margin-top: 80px;
+}
+
+
+/* Section */
+
+.section-items {
+    width: 100vw;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    padding-bottom: 60px;
+}
+
+.section-item {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;    
+}
+
+.section-item img {
+    width: 160px;
```

Comment:
Should not be pixels -- use relative measurements where possible (need to be careful if they're not svg files, in case the relative measurement starts to be bigger than the actual image size, resulting in blurring)